tests/conftest.py

```diff
@@ -2938,3 +2943,28 @@ def smbios_from_kubevirt_config(kubevirt_config_scope_module):
 @pytest.fixture(scope="session")
 def nmstate_required(admin_client):
     return get_cluster_platform(admin_client=admin_client) in ("BareMetal", "OpenStack")
+
+
+@pytest.fixture(scope="package")
+def enabled_aaq_in_hco_scope_package(admin_client, hco_namespace, hyperconverged_resource_scope_package):
+    with enable_aaq_in_hco(
+        client=admin_client,
+        hco_namespace=hco_namespace,
+        hyperconverged_resource=hyperconverged_resource_scope_package,
+    ):
+        yield
+
+
+@pytest.fixture(scope="module")
+def updated_namespace_with_aaq_label(admin_client, namespace):
+    label_project(name=namespace.name, label=AAQ_NAMESPACE_LABEL, admin_client=admin_client)
+
+
+@pytest.fixture(scope="class")
```

Comment:
you don't use classes in your test file. If you want to create arq once for all tests - use "module" scope

Reply:
with class scope this fixture will be executed every time for every test in your file since you don't have class in your test file

so either combine all tests under one class
or
change scope to module and in this case it will be executed only once